### ISSUE TITLE
update error: missing path

### ISSUE THREAD

**Issue by Adezandee** [angular/webdriver-manager#39]
Hi, 
An AssertionError pops with a fresh installation v10.0.2:

``` bash
$ npm i -g webdriver-manager
/usr/local/bin/webdriver-manager -> /usr/local/lib/node_modules/webdriver-manager/bin/webdriver-manager
webdriver-manager@10.0.2 /usr/local/lib/node_modules/webdriver-manager
├── q@1.4.1
├── adm-zip@0.4.7
├── minimist@1.2.0
├── chalk@1.1.3 (escape-string-regexp@1.0.5, ansi-styles@2.2.1, supports-color@2.0.0, has-ansi@2.0.0, strip-ansi@3.0.1)
└── request@2.72.0 (tunnel-agent@0.4.3, aws-sign2@0.6.0, forever-agent@0.6.1, oauth-sign@0.8.2, is-typedarray@1.0.0, caseless@0.11.0, stringstream@0.0.5, isstream@0.1.2, aws4@1.4.1, json-stringify-safe@5.0.1, extend@3.0.0, tough-cookie@2.2.2, node-uuid@1.4.7, qs@6.1.0, mime-types@2.1.11, form-data@1.0.0-rc4, combined-stream@1.0.5, har-validator@2.0.6, hawk@3.1.3, bl@1.1.2, http-signature@1.1.1)

$ webdriver-manager update
webdriver-manager: using global installed version 10.0.2
[10:58:29] E/config - nothing to return for a config file

assert.js:90
  throw new assert.AssertionError({
  ^
AssertionError: missing path
    at Module.require (module.js:466:3)
    at require (internal/module.js:20:19)
    at Function.Config.binaryVersions (/usr/local/lib/node_modules/webdriver-manager/built/lib/config.js:82:26)
    at /usr/local/lib/node_modules/webdriver-manager/built/lib/binaries/chrome_driver.js:47:51
    at Object.<anonymous> (/usr/local/lib/node_modules/webdriver-manager/built/lib/binaries/chrome_driver.js:51:2)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

A config file is required ? Thanks

**Comment by cnishina** [comment]
Noted. For now, use webdriver-manager as installed from protractor with `npm i -g protractor`. I'll have a fix for this soon.

**Comment by cnishina** [comment]
Closing this issue, it should be fixed with d3724fbd9f6b0ceb481538f7f8f0088c8b004959.

Released hot fix 10.0.3 for this. Please open a new issue if you are still seeing this problem.
